### ISSUE TITLE
How does GhostDB compare to ...

### ISSUE THREAD

**Issue by adamkennedy** [jakekgrog/GhostDB#12]
There are many KV stores, and many caches, and many in-memory things.

How does GhostDB compare? Can you add comparisons to memcache, redis, annakv.. etc to the website? Seems like that would be a FAQ.

**Comment by jakekgrog** [comment]
Yes, we'll add this! We're currently working on updating our docs and when that's done we'll publish to the website also.
